### ISSUE TITLE
Difference in README.md and observed software

### ISSUE THREAD

**Issue by florian-besser** [statping-ng/statping-ng#275]
Statping-ng advertises the very nice view e.g. in [this picture](https://camo.githubusercontent.com/348551d27da381eae52e3059ef69dd868fa7005521136138f57580d89f8b2f6d/68747470733a2f2f7374617470696e672d6e672e6769746875622e696f2f6173736574732f65787465726e616c2f7374617475707363322e706e67):

Namely it shows uptime last 24h and uptime since (I guess) forever. I am trying to replicate this behavior.

I have played with the [official Docker image](https://hub.docker.com/r/adamboutcher/statping-ng/tags) as well as built the `dev` branch myself using `docker build`. In both cases I get screenshots like below:
![image](https://github.com/statping-ng/statping-ng/assets/7090706/ed8dbad8-64e0-44b8-b84a-3e0dd546bf64)

Basically my screenshots show "Uptime last" twice, without any specifics. I assume it is meant to show "Uptime last 24h / last month / ???" but somehow the software bugs out and does not do so.

Any help on how to get that working?

**Comment by jemand771** [comment]
that should _not_ be happening on a build from the dev branch. it's a known previous issue that _should_ be fixed - #58 / #65 

try the new image: https://github.com/statping-ng/statping-ng/pkgs/container/statping-ng (`ghcr.io/statping-ng/statping-ng:0.91.0`)
I know moving registries is always a bit of a pain, but having the images here on github feels the most.. neatly integrated

**Comment by florian-besser** [comment]
Thanks for your answer. To verify I ran
```
docker run -it -p 8080:8080 ghcr.io/statping-ng/statping-ng:0.91.0
```

Unfortunately the result is the same:
![image](https://github.com/statping-ng/statping-ng/assets/7090706/30c6aec9-ac56-434c-9723-17e8c1e8867c)

I have not configured the software in any way except going through the setup when first hitting `http://0.0.0.0:8080/`:
![image](https://github.com/statping-ng/statping-ng/assets/7090706/81169b4f-e6cf-46ca-ac17-45413641d44f)

As such I believe I don't have any "bad data" laying around or anything else that should influence the software in question.

**Comment by jemand771** [comment]
I get a working screen with the exact same command 😭 
![image](https://github.com/statping-ng/statping-ng/assets/19669567/ca334f5c-0cc4-4979-9469-1ce208996503)

do you see anything interesting in the browser or container logs? what's your system language set to? (iirc this was an issue with translations)

**Comment by florian-besser** [comment]
Ah, thank you for your comment. It made me dig a bit deeper.

Good news: This is **working on Chrome**, but the error is present on **Firefox**.

I'm not sure anything useful will be in the network requests, but I recorded them anyway.

From Firefox:
[0.0.0.0_Archive [24-03-01 09-34-09].har.zip](https://github.com/statping-ng/statping-ng/files/14455393/0.0.0.0_Archive.24-03-01.09-34-09.har.zip)

From Chrome
[0.0.0.0_chrome.har.zip](https://github.com/statping-ng/statping-ng/files/14455400/0.0.0.0_chrome.har.zip)

As for language settings: My Firefox is set to English (US), with the system being set to English (Singapore) - historically Singapore follows British English more than US English if that's any help.
![Screenshot 2024-03-01 at 9 36 26 AM](https://github.com/statping-ng/statping-ng/assets/7090706/c70ad2bd-d578-4bee-808a-28df1c08cc3e)

I'm a backend developer by trade, so I'm unsure how to best help with what looks like a frontend issue - but if I can help somehow please let me know.

**Comment by jemand771** [comment]
what firefox version are you running? I only had Firefox ESR (115.8.0esr) on hand and it looks fine there

**Comment by florian-besser** [comment]
Running 123.0 (64-bit) here
![image](https://github.com/statping-ng/statping-ng/assets/7090706/e3ee4c8c-c5cf-413b-80e0-6ec7ff49d238)

**Comment by florian-besser** [comment]
I believe this issue is related to https://github.com/statping-ng/statping-ng/issues/283; I have since updated to Firefox `124.1` but I believe the issue was in cached data from an earlier run of `statping`.

Basically before discovering `statping-ng` I was playing around with `statping`. I was using the same host (`localhost`) as well as the same port (`8080`), thus I guess my browser was unaware of the switch / cached data!

This would also explain the "fix" of using Chrome: I had not used Chrome when playing around with `statping`, I only started using Chrome while debugging this issue. As Chrome had no cache present it worked as it should, with only Firefox having issues.

My apologies!
